dbms/src/DataTypes/DataTypeLowCardinalityHelpers.cpp

```diff
@@ -92,6 +92,9 @@ ColumnPtr recursiveLowCardinalityConversion(const ColumnPtr & column, const Data
     if (from_type->equals(*to_type))
         return column;
 
+    if (WhichDataType(to_type).isEnum() && from_type->getTypeId() == to_type->getTypeId())
```

Comment:
Need comment here.

Reply:
Also it's not correspond to function name now. Is is used only once. Maybe rename it?